UI/src/view/point_asset/massTransitStopForm.js

```diff
@@ -806,9 +897,14 @@
         var allProperties = selectedMassTransitStopModel.getProperties();
         var properties;
 
-        if (isTerminalBusStop) {
+        /* don't change == to ===
+        * sometimes the value is int other times is string*/
+        if (busStopTypeSelected == 6 ) {
           properties = sortAndFilterTerminalProperties(allProperties);
-        } else {
+        } else if  (busStopTypeSelected == 7){
```

Comment:
Same as above

Reply:
@marquesfm Check modification on this code